main.py

```diff
@@ -1,7 +1,46 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
+
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"),
+              ("Will", "Smith"),
+              ("Jaden", "Smith"),
+              ("Scarlett", "Johansson")]
+    # Create
```

Comment:
It is a redundant comment because it doesn't provide additional information. It is obvious what your code is doing here. Make logic separation with blank lines, that's it.
Fix everywhere